src/views/components/radios/index.njk

```diff
@@ -0,0 +1,56 @@
+---
+title: Radios
+description: Radios let the user select a single option from a list.
+layout: default
+eleventyNavigation:
+  key: Radios
+  section: components
+---
+{% from "example.njk" import example %}
+
+<section>
+  <p>Radios let the user select a single option from a list.</p>
+  {{ example({group: "components", item: "radios", example: "default", html: true, open: false }) }}
+</section>
+
+<section class="my-8">
+  <h2 class="mb-4">When to use this component</h2>
+  <p>Use the radio component when you need to help users select a single option from a list.</p>
+</section>
+
+<section class="mb-8">
+  <h2 class="mb-4">When not to use this component</h2>
+  <p>Do not use the radios component if users can choose more than one option from a list. In this case, use the <a href="/components/checkbox">checkboxes</a> component.</p>
+</section>
+
+<section class="mb-8">
+  <h2 class="mb-4">How to use this component</h2>
+  <p>Radios are grouped within a <code>fieldset</code> which starts with a <code>legend</code> which lets the user know what they need to do.</p>
+</section>
+
+<section class="mb-8">
+  <h2 class="mb-4">Variants</h2>
+
+  <h3 class="mb-4">Descriptions</h3>
+  <p>This variant allows you to add more context to the options in the list. The description is populated by providing a <code>description</code> parameter within the radio’s <code>label</code> parameter.</p>
```

Comment:
I think until you look at the example it is not clear that you need to add `with-description` rather than just `description`. Perhaps it'd also be handy to have a link to the `label` variant?

```suggestion
  <h3 class="mb-4">With description</h3>
  <p>This variant allows you to add more context to the options in the list. The description is populated by providing a <code>with-description</code> parameter within the radio’s <code>label</code> parameter.</p>
```

Reply:
That's a good suggestion, I'll update to be 'with description' and link to the label component